### PR TITLE
Fixed non-UTF8 characters.

### PR DESCRIPTION
**Pull request by fjpanag**
## Summary

Fixed several non-UTF8 characters throughout the source code.

See also [here](https://github.com/apache/incubator-nuttx/pull/7201).

## Impact

N/A

## Testing

N/A

### CODE REVIEW

**Comment by acassis** [comment]
Hi @fjpanag could you please also fix the issue that was already in these files and were reported by our CI, they are old files, before CI/nxstyle and need to be fixed

**Comment by fjpanag** [comment]
> Hi @fjpanag could you please also fix the issue that was already in these files and were reported by our CI, they are old files, before CI/nxstyle and need to be fixed

Yes of course. I run out of time yesterday...

**Review by xiaoxiang781216 — APPROVED** [review]

File: netutils/thttpd/timers.c [hunk 1]
```diff
@@ -293,12 +297,13 @@ void tmr_run(struct timeval *now)
...
-          /* Since the lists are sorted, as soon as we find a timer  * that isn'tmr
-           * ready yet, we can go on to the next list
+          /* Since the lists are sorted, as soon as we find a timer that
+           * isn't ready yet, we can go on to the next list.
            */
 
           if (tmr->time.tv_sec > now->tv_sec ||
-              (tmr->time.tv_sec == now->tv_sec && tmr->time.tv_usec > now->tv_usec))
+              (tmr->time.tv_sec == now->tv_sec &&
+               tmr->time.tv_usec > now->tv_usec))
```
Comment (pkarashchenko) [inline netutils/thttpd/timers.c:306]: what was an issue here?

File: netutils/thttpd/tdate_parse.c [hunk 2]
```diff
@@ -93,11 +93,14 @@ static int strlong_compare(const void *v1, const void *v2)
 #ifdef HAVE_DAY_OF_WEEK /* Day of week not yet supported by NuttX */
 static int strlong_search(char *str, struct strlong *tab, int n, long *lP)
```
Comment (pkarashchenko) [inline netutils/thttpd/tdate_parse.c:94]: maybe warning can be fixed?

File: netutils/thttpd/timers.c (hunk 1 above)
Comment (fjpanag) [reply netutils/thttpd/timers.c:306]: I got a warning for a long line.

File: netutils/thttpd/tdate_parse.c (hunk 2 above)
Comment (fjpanag) [reply netutils/thttpd/tdate_parse.c:94]: I didn't go through the code, so I don't know what `lP` means or what would be a more appropriate name.
Do you have a suggestion?

Comment (pkarashchenko) [reply netutils/thttpd/tdate_parse.c:94]: np. we can keep as is. Or just go to lower case `lp`

**Pull request merged**